src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -119,14 +119,51 @@
  * &lt;module name="NPathComplexity"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() { // NPath Complexity = 3 (OK)
+ *   if (a &gt; 10) {
+ *     if (a &gt; b) { // +1
+ *       baz();
+ *     }
+ *     else {  // +1
+ *       bar();
+ *     }
+ *   }
+ *   else { // +1
+ *     buzz();
+ *   }
+ * }
+ * </pre>
+ * <p>
  * To configure the check with a threshold of 1000:
  * </p>
  * <pre>
  * &lt;module name="NPathComplexity"&gt;
  *   &lt;property name="max" value="1000"/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() { // NPath Complexity = 2^10 = 1024 (violation, function exceeds 1000 paths)
```

Comment:
Please change it to ` // violation, NPath complexity is 1,024 (max allowed is 1,000)` and put it into the line where violation occurs.

Reply:
done